src/components/AboutContact.tsx

```diff
@@ -62,21 +72,53 @@ const AboutContact = () => {
                 <div className="detail-info">
                     <h1>{nameSurname}</h1>
                     <h3>Mobile</h3>
-                    <p>{mobile}</p>
+                    <img className="icon-details-info" src="/images/mobile-phone.svg" alt="icon" />
+                    <span>{mobile}</span> <br />
                     <h3>Email</h3>
-                    <p>{email}</p>
+                    <img className="icon-details-info" src="/images/email.svg" alt="icon" />
+                    <span>{email}</span>
+                    <h2>Job details</h2>
                     <h3>Position</h3>
-                    <p>{position}</p>
+                    <img className="icon-details-info" src="/images/work-case.svg" alt="icon" />
+                    <span>{position}</span> <br />
                     <h3>Job title</h3>
-                    <p>{jobTitle}</p>
-                    <h3>Social links</h3>
-                    <SocialIcon href={linkFacebook} src="facebook.svg" name="icon facebook" />
-                    <SocialIcon href={linkLinkedin} src="linkedin.svg" name="icon linkedin" />
-                    <SocialIcon href={linkTwitter} src="twitter.svg" name="icon twitter" />
-                    <h3>Export to CSV</h3>
-                    <SocialIcon href="#" src="importCSV.svg" name="icon import CSV" />
+                    <img className="icon-details-info" src="/images/location-company.svg" alt="icon" />
+                    <span>{jobTitle}</span>
+                    <div className="social-links">
+                        <h4>Social links</h4>
+                        <SocialIcon href={linkFacebook} src="facebook.svg" name="icon facebook" />
+                        <SocialIcon href={linkLinkedin} src="linkedin.svg" name="icon linkedin" />
+                        <SocialIcon href={linkTwitter} src="twitter.svg" name="icon twitter" />
+                    </div>
+                    <div className="export-svg">
+                        <h3>Export to CSV</h3>
+                        <img id="desktop-export" src='/images/exportCSV.svg' alt="SVG export" />
```

Comment:
Why do you use id here instead of class?

Reply:
I was trying to select a specific element <img> here, thought it is better to use id instead of className.
I changed it to class as everywhere I use class as well.